js/container.ts

```diff
@@ -0,0 +1,128 @@
+import type { RenderProps } from "@anywidget/types";
+import { css, html, TemplateResult } from "lit";
+import { property } from "lit/decorators.js";
+
+import { legacyStyles } from "./ipywidgets_styles";
+import { LitWidget } from "./lit_widget";
+import { materialStyles } from "./styles";
+
+export interface ContainerModel {
+    title: string;
+    collapsed: boolean;
+    hide_close_button: boolean;
+}
+
+export class Container extends LitWidget<ContainerModel, Container> {
+    static get componentName(): string {
+        return `widget-container`;
+    }
+
+    static styles = [
+        legacyStyles,
+        materialStyles,
+        css`
+            .header {
+                display: flex;
+                gap: 4px;
+                margin: 4px;
```

Comment:
Use `padding` instead of `margin` to prevent white border (and below in `widget-container`):
![image](https://github.com/user-attachments/assets/0fc8adc9-709d-46b0-8a27-165ee7d758b7)